osu.Game/Screens/Multiplayer/MultiRadioBox.cs

```diff
@@ -0,0 +1,159 @@
+﻿// Copyright (c) 2007-2017 ppy Pty Ltd <contact@ppy.sh>.
+// Licensed under the MIT Licence - https://raw.githubusercontent.com/ppy/osu/master/LICENCE
+
+using OpenTK.Graphics;
+using osu.Framework.Graphics;
+using osu.Framework.Graphics.Containers;
+using osu.Framework.Graphics.Primitives;
+using osu.Framework.Graphics.Sprites;
+using osu.Framework.Input;
+using osu.Game.Graphics.Sprites;
+using System;
+
+
+
+namespace osu.Game.Screens.Multiplayer
+{
+    public class MultiRadioBox : Container

```

Comment:
Please make a general abstract class similar to the `DropDownMenu<T>` already existing in the framework, and put it into the framework. Something like `TabMenu<T>`. Then you can implement that abstract class here to give it your specific design and tabs.